### PR TITLE
Update README file for 10.1 release

### PR DESCRIPTION
**Pull request by paresh-bhagat12**
Update build steps for 10.01.10 release

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,17 +94,20 @@ the external tree during the build process. To configure the system for build,
 choose any of the *defconfigs*  provided in this `BR2_EXTERNAL` tree. For example:
 
 ```bash
-For AM62X
-buildroot/ $ make BR2_EXTERNAL=../buildroot-external-TI ti_release_am62x_sk_defconfig
+For AM62X Linux
```
Comment (cshilwant) [inline README.md:97]:
```suggestion
# For AM62X Linux
```

Comment (cshilwant) [reply README.md:97]: better to mark it as comment, for bash to recognize & highlight

**Review by jeevantelukula — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -94,17 +94,20 @@ the external tree during the build process. To configure the system for build,
...
+$ make BR2_EXTERNAL=../buildroot-external-TI ti_release_am62x_sk_defconfig
+
+For AM62X RT-Linux
```
Comment (cshilwant) [inline README.md:100]:
```suggestion
# For AM62X RT-Linux
```
better to mark it as comment, for bash to recognize & highlight

**Review by jsuhaas22 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (paresh-bhagat12) [reply README.md]: Updated @cshilwant

**Review by cshilwant — APPROVED** [review]
LGTM now!

**Pull request merged**